serverless-plugin/tests/alarms-api-gateway.test.js

```diff
@@ -1,8 +1,8 @@
 'use strict'
 
-const apiGatewayAlarms = require('../alarms-api-gateway')
+const apiGatewayAlarms = require('../../core/alarms-api-gateway')
```

Comment:
These tests relate to modules in `core`, so the tests themselves can be moved to `core`. Then, you won't need such deep relative imports with `../..`

Reply:
updated